views/tx.tmpl

```diff
@@ -32,6 +32,28 @@
                         </td>
                     </tr>
                 {{end}}
+                {{if eq .Type "Vote"}}
+                    {{range .Vin}}
+                        {{if .Txid}}
+                            <tr>
+                                <td width="90" class="text-right pr-2 h1rem p03rem0 xs-w91">TICKET SPENT</td>
+                                <td><a href="/explorer/tx/{{.Txid}}">{{.Txid}}</a></td>
+                            </tr>
+                        {{end}}
+                    {{end}}
+                {{end}}
+                {{if eq .Type "Ticket"}}
+                    {{range $i, $v :=  .SpendingTxns}}
```

Comment:
Don't range.  You only need the spending transaction hash for the first output.

Reply:
Alright, that helps alot